tools/unitctl/unit-openapi/README.md

```diff
@@ -409,4 +409,3 @@ cargo doc --open
 ## Author
 
 unit-owner@nginx.org
-
```

Comment:
No.

Leave the blank line in there otherwise we're going to just be ping-ponging on this file in every PR. That's why we introduced the .`gitattributes` file in that directory.